src/components/ha-paper-slider.html

```diff
@@ -0,0 +1,80 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-slider/paper-slider.html">
+
+
+<dom-module id="ha-paper-slider">
+  <template strip-whitespace>
+    <style include="paper-slider">
+      .pin > .slider-knob > .slider-knob-inner {
+        font-size:  var(--ha-paper-slider-pin-font-size, 10px);
+        line-height: normal;
+      }
+
+      .pin > .slider-knob > .slider-knob-inner::before {
+        top: unset;
+        margin-left: unset;
+
+        bottom: calc(15px + var(--calculated-paper-slider-height)/2);
+        left: 50%;
+        width: 2.6em;
+        height: 2.6em;
+
+        -webkit-transform-origin: left bottom;
+        transform-origin: left bottom;
+        -webkit-transform: rotate(-45deg) scale(0) translate(0);
+        transform: rotate(-45deg) scale(0) translate(0);
+      }
+
+      .pin.expand > .slider-knob > .slider-knob-inner::before {
+        -webkit-transform: rotate(-45deg) scale(1) translate(7px, -7px);
+        transform: rotate(-45deg) scale(1) translate(7px, -7px);
+      }
+
+      .pin > .slider-knob > .slider-knob-inner::after {
+        top: unset;
+        font-size: unset;
+
+        bottom: calc(15px + var(--calculated-paper-slider-height)/2);
+        left: 50%;
+        margin-left: -1.3em;
+        width: 2.6em;
+        height: 2.4em;
+
+        -webkit-transform-origin: center bottom;
+        transform-origin: center bottom;
+        -webkit-transform: scale(0) translate(0);
+        transform: scale(0) translate(0);
+      }
+
+      .pin.expand > .slider-knob > .slider-knob-inner::after {
+        -webkit-transform: scale(1) translate(0, -10px);
+        transform: scale(1) translate(0, -10px);
+      }
+    </style>
+  </template>
+
+  <script>
+    customElements.whenDefined('paper-slider').then(() => {
+      const PaperSliderClass = customElements.get('paper-slider');
+      let myTemplate;
+      class HaPaperSlider extends PaperSliderClass {
+        static get is() { return 'ha-paper-slider'; }
+
+        static get template() {
+          if (!myTemplate) {
+            // Retrieve this element's dom-module template
+            myTemplate = Polymer.DomModule.import(this.is, 'template');
```

Comment:
I think `HaPaperSlider.is` is more clear than `this.is` in a static context.

Reply:
In this case the target is actually to get the parents class (PaperSlider) template. 
So I think then it it would be more clear to use `PaperSlider.is`

P.S. I like little discussions like this, I learn a lot from them 👍

Reply:
You are right. I responded without taking a good look at the code. Fixed.